packaging/installer/kickstart.sh

```diff
@@ -84,9 +84,15 @@ main() {
     exit 0
   fi
 
-  tmpdir="$(create_tmp_directory)"
-  progress "Using ${tmpdir} as a temporary directory."
-  cd "${tmpdir}" || fatal "Failed to change current working directory to ${tmpdir}." F000A
+  tempfoo=`basename -s .sh $0`
+  # mktemp behaves slightly differently on macOS but we ignore this fact...
+  wrkdir=`mktemp -q -d -t ${tempfoo}.XXXXX`
```

Comment:
We can’t just blindly run `mktemp` and assume things will work. The working directory needs to allow both writing files _and_ executing files, and we actually need to confirm that this is the case (we have had significant issues in the past relating to this).

Handling that correctly, and falling back properly, was the whole point of the `create_tmp_directory` code.